VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -59,12 +61,17 @@ public async Task AddToGroup(string conferenceId)
         }
     }
     
-    private async Task AddUserToUserGroup(bool isAdmin)
+    private async Task AddUserToUserGroup(bool isAdmin, bool isStaffMember)
     {
         if (isAdmin)
         {
             await Groups.AddToGroupAsync(Context.ConnectionId, VhOfficersGroupName);
         }
+
+        if (isStaffMember)
```

Comment:
because we only message VHOs and participants who are part of the hearing. A staff member who is opening the list for monitoring may not be part of the hearing, and so they wouldn't receive the message